containers/nginx-storage/default.conf

```diff
@@ -0,0 +1,12 @@
+error_log logs/error.log notice;
+
```

Comment:
`default.conf` and `file-transfer-config.lua` should be added in a separate commit. Basically, creating the default, hard-coded config for the image is a separate change and should have its own commit(s).

Reply:
Done.